torch/distributed/distributed_c10d.py

```diff
@@ -2231,6 +2231,15 @@ def append(self, work: Work):
     def wait(self):
         for work in self.works:
             work.wait()
+# Global set to store coalescing supported device types
+COALESING_SUPPORTED_DTYPES = {"cuda", "hpu"}
```

Comment:
nit: typo? COALESING -> COALESCING
nit: dtype or device type? (the former is often understood as data type)

Reply:
nit: prefix this variable with "_" ?